### ISSUE TITLE
Distinguishing between arrangedSubviews and general subviews in the sub layout of the UIStackView

### ISSUE THREAD

**Issue by eugeneShin-dev** [ioskrew/SwiftLayout#238]
In the versions before iOS14, in order to set background color of UIStackView, an extra UIView should be added.

**Comment by gmlwhdtjd** [comment]
@eugeneShin-dev 
This issue will be fixed in the next version

```swift
let stackView = UIStackView()
let notArrangedview1 = UIView()
let notArrangedview2 = UIView()
let arrangedView1 = UIView()
let arrangedView2 = UIView()
let arrangedView3 = UIView()

@LayoutBuilder
var layout: some Layout {
    root.sublayout {
        stackView.sublayout {
            GroupLayout(option: .isNotArranged) {
                notArrangedview1
                notArrangedview2
            }

            arrangedView1
            arrangedView2
            arrangedView3
        }
    }
}
```

**Comment by eugeneShin-dev** [comment]
수정감사합니다!

**Comment by gmlwhdtjd** [comment]
@eugeneShin-dev 
It is available in 2.7.0.
